### ISSUE TITLE
Init produces duplicate blank lines on Windows

### ISSUE THREAD

**Issue by wilt00** [dmbaturin/soupault#19]
Running `soupault --init` on Windows (10, build 19041.685) generates a file with numerous blank lines:

![image](https://user-images.githubusercontent.com/7883508/104137254-1984d180-5361-11eb-855f-1af0874b875f.png)

Note that the file encoding in the lower right is detected as "Macintosh (CR)"

Looking at the hex dump shows what's happening:

![image](https://user-images.githubusercontent.com/7883508/104137272-420ccb80-5361-11eb-83a1-5d6d2cd0e2e7.png)

Each line ends with `\r\r\n`, rather than `\r\n`. It seems that the LF -> CRLF transformation is being applied more than once.

**Comment by wilt00** [comment]
I should have mentioned, this is true of main.html and index.html as well, not just soupault.conf.

**Comment by dmbaturin** [comment]
Interesting, looks like I goofed out somewhere. Are you building from source or using an official build (i.e. will you need me to make a test build for you to verify the fix?).

**Comment by wilt00** [comment]
Sorry, this is the 2.3.0 build from https://soupault.neocities.org/#downloads . I'm working on setting up opam to build from source now, will report back if that still has the bug.

**Comment by dmbaturin** [comment]
I'm pretty sure if that bug exists in 2.3, it exists in the latest code as well—I'll look into it. My Windows build setup is [fd-open's installer](https://fdopen.github.io/opam-repository-mingw/installation/) on Windows Server 2019.

**Comment by wilt00** [comment]
Are you using git to get the code onto the windows build server? It could be that git for windows's `core.autocrlf` setting is transforming line endings to CRLF on checkout, which I think it does by default.

**Comment by dmbaturin** [comment]
Yes, I', using git in cygwin. However, it looks like I just didn't give the OCaml runtime enough credit: it seems to be capable of producing correct newlines for the host OS by itself just fine!

I've set `core.autocrlf` to false by hand to rule out that issue, but that didn't change anything. However, when I removed the fixup for newlines I thought would be needed, extra newlines were gone. I've made a test build: https://files.baturin.org/tmp/soupault-2.4.0-dev+fix19.exe (sha1=b1db6ae82f32cd8650f4072ce43e097669032951)
